micro-rdk/src/esp32/board.rs

```diff
@@ -68,6 +73,9 @@ impl EspBoard {
     /// The potential approach is described in esp32/motor.rs:383
     pub(crate) fn from_config(cfg: ConfigType) -> Result<BoardType, BoardError> {
         let (analogs, mut pins, i2c_confs) = {
+            #[cfg(not(esp32))]
```

Comment:
This needs a ticket and a TODO added. There will be no analog pin support on esp32s3 until this is fixed.